charts/kubescape-operator/templates/kubescape/servicemonitor.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
   labels:
     app: {{ .Values.kubescape.name }}
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    {{- with .Values.kubescape.serviceMonitor.labels }}
```

Comment:
```suggestion
    {{- with .Values.kubescape.labels }}
```

Reply:
These are labels specifically for serviceMonitor, as they are only needed for prometheus-operator to work correctly. It seems to me that common labels will not work here

We can add separately `labels` or `commonLabels` for common cases.

Reply:
@dwertent What do you think?